docs/core/compatibility/core-libraries/6.0/static-abstract-interface-methods.md

```diff
@@ -3,7 +3,7 @@ title: "Breaking change: Static abstract members in interfaces"
 description: Learn about the .NET 6 breaking change where `static` interface members can now be marked `abstract`.
 ms.date: 09/08/2021
 ---
-# Static abstract members in interfaces
+# Static abstract members declared in interfaces
```

Comment:
ping @gewarren Is this change OK? I need to update the title for this PR to get a clean build. The feature spec in the C# 11 folder has the same H1.